lib/shared/log/tasks.js

```diff
@@ -71,55 +77,65 @@ function logTasks(tree, opts, getTask) {
   }
 
   function addTaskToLines(task, lines, isLast, isLeaf) {
-    var taskBars = task.bars + (isLast ? '└' : '├') + '─';
+    var taskBars = task.bars + (isLast
+        ? translate.message(messages.BOX_DRAWINGS_LIGHT_UP_AND_RIGHT)
+        : translate.message(messages.BOX_DRAWINGS_LIGHT_VERTICAL_AND_RIGHT)) +
+          translate.message(messages.BOX_DRAWINGS_LIGHT_HORIZONTAL);
     if (isLeaf) {
-      taskBars += '─ ';
+      taskBars += translate.message(messages.BOX_DRAWINGS_LIGHT_HORIZONTAL);
     } else {
-      taskBars += '┬ ';
+      taskBars += translate.message(messages.BOX_DRAWINGS_LIGHT_DOWN_AND_HORIZONTAL);
     }
+    taskBars += ' ';
 
     var line = {};
     if (task.depth === 1) {
-      line.label = chalk.white(taskBars) + chalk.white(task.label);
+      line.label = taskBars + task.label
```

Comment:
Not allowing people to theme the task names since those should be verbatim for the gulpfile. They can name tasks them in a different language already and I've removed the coloring all together.

Reply:
This depends on https://github.com/gulpjs/gulp-cli/pull/260#discussion_r1528380052

Reply:
Can you explain?

Reply:
@sttk I've added more messages so I can translate this.